tjreports/plugins/system/plg_system_sendemail/bulksendemail.js

```diff
@@ -0,0 +1,282 @@
+																modelEmail += Joomla.JText._('PLG_SYSTEM_SENDEMAIL_POPUP_EMAIL_SUBJECT') + ' <span class="star">&nbsp;*</span>';
+															modelEmail += '</label>';
+														modelEmail += '</div>';
+
+														modelEmail += '<div class="controls">';
+															modelEmail += '<input type="text" name="template[subject]" id="' + emailSubject + '" value="" class="required w-100" required="required" aria-required="true">';
+														modelEmail += '</div>';
+													modelEmail += '</div>';
+													modelEmail += '<div class="control-group">';
+														modelEmail += '<div class="control-label">';
+															modelEmail += '<label id="email-message-label" for="email-message" class="hasPopover required" title="" >';
+																modelEmail += Joomla.JText._('PLG_SYSTEM_SENDEMAIL_POPUP_EMAIL_BODY_MESSAGE') + ' <span class="star">&nbsp;*</span>';
+															modelEmail += '</label>';
+														modelEmail += '</div>';
+														modelEmail += '<div class="controls">';
+															modelEmail += '<textarea name="template[message]" rows="7" style="width: 30%;" id="'+ emailMessage + '" value="" class="required" required="required" aria-required="true"></textarea>';
+														modelEmail += '</div>';
+													modelEmail += '</div>';
+												modelEmail += '</div>';
+												modelEmail += '<div id="emailsDiv"></div>';
+											modelEmail += '</div>';
+										modelEmail += '</div>';
+									modelEmail += '</form>';
+								modelEmail += '</div>';
+								modelEmail += '<div class="modal-footer">';
+									modelEmail += '<button type="button" class="btn btn-primary validate" id="send-email" onclick="tjutilitysendemail.validate();">' + Joomla.JText._('PLG_SYSTEM_SENDEMAIL_POPUP_SEND_BTN') + '</button>';
+								modelEmail += '</div>';
+						modelEmail += '</div>';
+					modelEmail += '</div>';
+				modelEmail += '</div>';
+
+			// Confirm this class to append popup
+			jQuery('#j-main-container').append(modelEmail);
+
+			jQuery('div').find("#preload").hide();
+			jQuery('div').find(".is-progress").removeClass('is-progress');
+			jQuery('div').find("#send-email").attr("disabled", false);
+			jQuery("#" + emailSubject).val('');
+			jQuery("#" + emailMessage).val('');
+
+			var values = new Array();
+			jQuery("#emailsDiv").empty();
+
+			jQuery.each(jQuery("input[name='cid[]']:checked").closest("td").siblings("td" + tjutilitysendemail.tjTdClass), function () {
+				values.push(jQuery(this).text());
+
+				var hiddenEle = "<input readonly type='hidden' name='emails[]' value='" + jQuery(this).text() + "'/>";
+				jQuery("#bulkEmailModal").find("#emailsDiv").append(hiddenEle);
+			});
+
+			// Create editor field
+			 tinyMCE.execCommand( 'mceAddEditor', false, "email-message");
+
+			// alert(values.join (", "));
+		}
+		catch (err) {
+			console.log(err.message);
```

Comment:
Show alert and log error in the console under error

Reply:
console.error API

Reply:
Sir I didn't understand console.errorAPI so only gave alert message now.